daemon/algod/api/server/v2/handlers.go

```diff
@@ -781,6 +798,14 @@ func (v2 *Handlers) GetStatus(ctx echo.Context) error {
 		CatchpointAcquiredBlocks:    &stat.CatchpointCatchupAcquiredBlocks,
 	}
 
+	if upgradePropose != "" {
```

Comment:
This also needs to check that the vote has not finished:
```suggestion
	if upgradePropose != "" && latestBlkHdr.NextProtocolVoteBefore > stat.LastRound {
```

Reply:
done